google/cloud/internal/oauth2_service_account_credentials_test.cc

```diff
@@ -873,6 +686,7 @@ TEST(ServiceAccountCredentialsTest, ParseServiceAccountRefreshResponse) {
   ScopedEnvironment disable_self_signed_jwt(
       "GOOGLE_CLOUD_CPP_EXPERIMENTAL_DISABLE_SELF_SIGNED_JWT", "1");
 
+  auto const expires_in = 1000;
```

Comment:
Make it a `std::chrono` duration?

Reply:
Done, though I have been avoiding too many test cleanups in these changes.